woocommerce/woocommerce-ios
# [Mobile Payments] Announce Card Present Payments feature on Payment Methods screen

Closes: #7229 


### Description


Adds a banner for the first iteration of upselling Card Present Payments in the app. This banner is on the Payment Method screen, and is permanently shown: we can look to remove it in future when it is no longer useful, but for now there is plenty of space and it is below all the actions on that screen.

This banner only shows for stores in countries where Card Present Payments is supported.


### Testing instructions


1. Launch the app and go to the Orders tab
2. Tap `+` and select `Simple Payment` or `Create Order
3. Set up the payment/order and go through to the payment method selection screen.
4. If your store is in the US or Canada, observe that the banner is shown. Otherwise, observe that it isn't, and stop.
5. Tap on the CTA
6. Observe that the web view opens and loads the correct product page for your store's country: M2 reader for US, WisePad for Canada
7. Observe that you can tap buy, log in, and get to the checkout.

### Screenshots


https://user-images.githubusercontent.com/2472348/177575409-b98d1fbe-0944-4aac-9aa3-3dde594f8dc6.mp4



---
